src/test/java/com/jadajabajava/JobSeekerServiceTest.java

```diff
@@ -0,0 +1,11 @@
+package com.jadajabajava;
+
+import org.junit.jupiter.api.Test;
+
+public class JobSeekerServiceTest {
+
+    @Test
+    public void someTest() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/gh/jada-jaba-java/job-seeker-service/pullRequest?prid=5692048)